sdk/storage_blobs/examples/container_00.rs

```diff
@@ -23,30 +23,26 @@ async fn main() -> azure_core::Result<()> {
     let container_client = storage_client.as_container_client(container_name);
 
     let max_results = NonZeroU32::new(3).unwrap();
-    let mut iv = Box::pin(
-        blob_service_client
-            .list_containers()
-            .max_results(max_results)
-            .stream(),
-    );
+    let mut iv = blob_service_client
+        .list_containers()
+        .max_results(max_results)
+        .into_stream();
 
     let mut count = 0;
     while let Some(result) = iv.next().await {
         let container = result?;
-        count += container.incomplete_vector.len();
-        for container in container.incomplete_vector.iter() {
+        count += container.containers.len();
```

Comment:
nit: `container.containers` reads a bit weird. Perhaps the `container` binding should be renamed to `page` or `response`?